source/integrate-with-integration-environment/prove-users-identity.html.md.erb

```diff
@@ -296,7 +296,7 @@ Each JSON object in the list may contain any of the following properties:
 | `addressLocality`                | Maps to `POST_TOWN` in the [Postcode Address File][external.postcode-file] and [Ordnance Survey Places API][external.os-places-api].                                                                                                                                                                                          |
 | `postalCode`                     | Maps to `POST_CODE` in the [Postcode Address File][external.postcode-file] and [Ordnance Survey Places API][external.os-places-api].                                                                                                                                                                                          |
 | `addressCountry`                 | Two-letter [ISO 3166-1 alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2).                                                                                                                                                                                                                               |
-
+| `addressRegion`                  | Maps to [schema:addressRegion](https://schema.org/addressRegion). Only returned for international addresses and will contain the region, provided as text. For example, California or another appropriate first-level Administrative division.                                                                                |
```

Comment:
Looks good to me, just querying if the capital A is necessary on Administrative division - I've had a google and I've seen it lower case and upper, is there a reason why you opted for upper case?

Reply:
That's how it's referenced on the schema.org page: https://schema.org/addressRegion, which is where the info for this column comes from.